ui/uihandler.go

```diff
@@ -129,47 +139,41 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		}
 		w.Header().Set("Content-Type", "text/html; charset=UTF-8")
 		err := mainTemplate.Execute(w, &struct {
-			R           *http.Request
-			Headers     http.Header
-			Version     string
-			LogoPath    string
-			DebugPath   string
-			ChartJSPath string
-			StartTime   string
-			TargetURL   string
-			Labels      string
-			UpTime      time.Duration
-			Port        int
-			DoExit      bool
-			DoLoad      bool
+			R                           *http.Request
+			Headers                     http.Header
+			Version                     string
+			LogoPath                    string
+			DebugPath                   string
+			ChartJSPath                 string
+			StartTime                   string
+			TargetURL                   string
+			Labels                      string
+			UpTime                      time.Duration
+			Port                        int
+			DoExit                      bool
+			DoLoad                      bool
+			TestExpectedDurationSeconds float64
```

Comment:
surprised you didn't get some alignment/sizeof struct linter there, might need to move expected durations up before the bools

Reply:
done!